components/strava/strava.app.js

```diff
@@ -1,21 +1,36 @@
 // Strava API app file
 const axios = require("axios");
-
+const { axios: PlatformAxios } = require("@pipedream/platform");
 module.exports = {
   type: "app",
   app: "strava",
   methods: {
-    async _makeAPIRequest(opts) {
+    prepareOpts(opts) {
       if (!opts.headers) opts.headers = {};
       opts.headers["Authorization"] = `Bearer ${this.$auth.oauth_access_token}`;
       opts.headers["Content-Type"] = "application/json";
       opts.headers["user-agent"] = "@PipedreamHQ/pipedream v0.1";
       const { path } = opts;
       delete opts.path;
       opts.url = `https://www.strava.com/api/v3${
-        path[0] === "/" ? "" : "/"
+        path[0] === "/" ?
+          "" :
+          "/"
       }${path}`;
-      return await axios(opts);
+      return opts;
+    },
+    makeQueryString(params) {
```

Comment:
`axios` should accept a `params` object, from which they create a query string. So you shouldn't need this method, and you should be able to pass any query params in `params`. Let me know if that works

Reply:
I deleted the method and used params object. It works well